src/commands/login.ts

```diff
@@ -32,11 +37,20 @@ async function getTagoDeployURL(): Promise<{ urlAPI: string; urlSSE: string } |
   }
 
   if (urlSSE) {
+    const isFullURLSSE = urlSSE.includes("https://sse.");
+    if (!isFullURLSSE && !urlSSE.includes("sse.")) {
+      urlSSE = urlSSE.replace("api.", "sse.");
+      urlSSE = `https://${urlSSE}`;
+    }
+
```

Comment:
If goal is to add https://, you should make a function for it and call for both situations.

Also, I don't see why verify for "sse.". The replace will only replace an "api." if it exists anyway.